content/association/_index.md

```diff
@@ -5,37 +5,119 @@ title = "Association"
 > Note:
 >
 > This page is a draft and is subject to change. The association is not yet registered
-> and we are still considering options on how and where it should be. To help us
-> develop these statutes or the association you can join us in the
-> [Biweekly Nordic RSE meeting](/communities/join/#bi-weekly-meetings).
-> 
-> Our suggestion at the moment is to register the association in Finland, since we know the relevant laws.
-> This is not necessarily an informed decision and we would welcome input about the possibility of registering
-> in a different Nordic country.
->
-> It should be possible for members, board members and the chair to not be citizens of the registration country.
-> Other than that, the registration process should be as straightforward as possible.
+> and we are still considering our options.
 >
 
 # The Association
 
-The Nordic RSE group is planning to register in Finland as the Research Software Engineers Ry.
+The Nordic RSE group is planning to register in Finland as the
+Nordic Research Software Engineers Ry.
 Registering as an association comes with certain legal benefits, such as the ability to
 sign contracts and own assets.
 Registering an association will also create a clear membership and governing
 procedures.
 
-The association is governed by an annual assembly of its members, which is held
-mainly online, and its day to day activities are taken care of by a board.
+The association is governed by an annual assembly of it's members, which is held
+mainly online, and it's day to day activities are taken care of by a board.
```

Comment:
```suggestion
mainly online, and its day to day activities are taken care of by a board.
```

Reply:
Also the other: it should be "its". GitHub does not let me line-suggest the other one.